controllers/cloud.redhat.com/providers/iqe/impl.go

```diff
@@ -35,12 +35,16 @@ func joinNullableSlice(s *[]string) string {
 
 func createIqeContainer(j *batchv1.Job, nn types.NamespacedName, cji *crd.ClowdJobInvocation, env *crd.ClowdEnvironment, app *crd.ClowdApp) *core.Container {
 	// create env vars
+	iqePlugins := app.Spec.Testing.IqePlugin
+	if cji.Spec.Testing.Iqe.IqePlugins != "" {
+		iqePlugins = cji.Spec.Testing.Iqe.IqePlugins
+	}
 	envVars := []core.EnvVar{
 		{Name: "ENV_FOR_DYNACONF", Value: cji.Spec.Testing.Iqe.DynaconfEnvName},
 		{Name: "NAMESPACE", Value: nn.Namespace},
 		{Name: "CLOWDER_ENABLED", Value: "true"},
 		{Name: "ACG_CONFIG", Value: "/cdapp/cdappconfig.json"},
-		{Name: "IQE_PLUGINS", Value: app.Spec.Testing.IqePlugin},
+		{Name: "IQE_PLUGINS", Value: iqePlugins},
```

Comment:
Same sort of question as above. Do you get your app's IQE plugin included or does it need to be explicitly added with your extra plugins?

Reply:
If you specify `plugins` on the CJI it totally overrides `app.Spec.Testing.IqePlugin`